File: projectBlueWater/src/main/java/com/lasthopesoftware/permissions/storage/write/request/IStorageWritePermissionsRequestNotificationBuilder.java
package com.lasthopesoftware.permissions.storage.write.request;

import android.app.Notification;

/**
 * Created by david on 7/3/16.
 */
public interface IStorageWritePermissionsRequestNotificationBuilder {
	Notification buildWritePermissionsRequestNotification(int libraryId);
}

File: projectBlueWater/src/main/java/com/lasthopesoftware/bluewater/client/browsing/library/request/write/IStorageWritePermissionsRequestNotificationBuilder.kt
package com.lasthopesoftware.bluewater.client.browsing.library.request.write;

import android.app.Notification;

/**
 * Created by david on 7/3/16.
 */
public interface IStorageWritePermissionsRequestNotificationBuilder {
	Notification buildWritePermissionsRequestNotification(int libraryId);
}
